chain-storage/src/error.rs

```diff
@@ -4,13 +4,15 @@ use std::{error, fmt};
 pub enum Error {
     BlockNotFound, // FIXME: add BlockId
     CannotIterate,
+    BackendError,
```

Comment:
Good. It could also pack a `Box<dyn Error + Send + Sync>` with the underlying error that would be chained in the `Error` impl, but this can be done later.

Reply:
Done, thanks!